projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -58,8 +59,14 @@ function register_block() {
 		return;
 	}
 
-	if ( ! Jetpack_Gutenberg::is_newsletter_enabled() ) {
-		return; // Stop here if the Newsletter feature is not enabled.
+	/**
+	 * Do not proceed if the newsletter feature is not enabled
+	 */
+	if (
+		/** This filter is documented in class.jetpack-gutenberg.php */
+		! apply_filters( 'jetpack_subscriptions_newsletter_feature_enabled', false )
+	) {
```

Comment:
Should we check if `class_exists( '\Jetpack_Memberships' )` here as well?

Reply:
Added in [78bc0d0](https://github.com/Automattic/jetpack/pull/29860/commits/78bc0d02308cce7625df82d774f73a353ae0e5d6)!